### PR TITLE
Update node version to 19

### PR DESCRIPTION
**Pull request by aaroncameron-wk**
Update dependencies and Dockerfile for node v19.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,17 @@
-FROM node:16-slim as node-build
+FROM node:19-slim as node-build
 
 ARG NPM_CONFIG__AUTH
 ARG NPM_CONFIG_REGISTRY=https://workivaeast.jfrog.io/workivaeast/api/npm/npm-prod/
 ARG NPM_CONFIG_ALWAYS_AUTH=true
 ARG GIT_TAG
 
+RUN reg=$(echo "$NPM_CONFIG_REGISTRY" | cut -d ":" -f 2) && \
```
Comment (brettkail-wk) [inline Dockerfile:8]: If this is what's required, it seems like we should report that to #support-build to let them know how tedious this is so they can, e.g., add an `NPMRC_CONTENT` env var that has the full contents of this file (ala `KNOWN_HOSTS_CONTENT`).

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+    echo "$reg:_auth = $NPM_CONFIG__AUTH" > /.npmrc && \
+    echo "registry = $NPM_CONFIG_REGISTRY" >> /.npmrc && \
+    echo "always-auth = true" >> /.npmrc
+ARG NPM_CONFIG_USERCONFIG=/.npmrc
```
Comment (brettkail-wk) [inline Dockerfile:12]: What is the default file (is it `/root/.npmrc`), and we just write to that instead so we don't need this extra env var?

Comment (aaroncameron-wk) [reply Dockerfile:12]: Definitely. There's some Rosie hijinks on this PR so I'll get this change included on my next ixbrl-viewer PR.

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**